gravitee-am-service/src/main/resources/flow/am-schema.json

```diff
@@ -27,7 +27,8 @@
           "login": "LOGIN",
           "consent": "CONSENT",
           "register": "REGISTER",
-          "reset_password": "RESET PASSWORD"
+          "reset_password": "RESET PASSWORD",
+          "registration_confirmation": "REGISTRATION CONFIRMATION"
```

Comment:
`registration_confirmation`

Needs also to be added to enums above.

Reply:
Also the unit test on the number of flows don't pass

Reply:
FP